src/data/faq_de.json

```diff
@@ -2042,7 +2042,7 @@
                                 "active": false,
                                 "textblock": [
                                     "<b>AKTUELL</b><br/> Das Problem ist in Version 1.7.1 behoben worden.<hr/>",
-                                    "Die Meldung <a href='https://github.com/corona-warn-app/cwa-app-ios/issues/1497' target='_blank'>'ExposureDetectionIsAlreadyRunning'</a> <a href='https://github.com/corona-warn-app/cwa-app-ios/issues/1512#issuecomment-727206633' target='_blank'>kann ignoriert werden</a>. Die CWA funktioniert korrekt und dieses Problem wird in einem <a href='https://github.com/corona-warn-app/cwa-app-ios/pull/1510' target='_blank'> bevorstehenden Update korrigiert</a>."
+                                    "Die Meldung <a href='https://github.com/corona-warn-app/cwa-app-ios/issues/1497' target='_blank'>'ExposureDetectionIsAlreadyRunning'</a> <a href='https://github.com/corona-warn-app/cwa-app-ios/issues/1512#issuecomment-727206633' target='_blank' rel='noopener noreferrer'>kann ignoriert werden</a>. Die CWA funktioniert korrekt und dieses Problem wird in einem <a href='https://github.com/corona-warn-app/cwa-app-ios/pull/1510' target='_blank' rel='noopener noreferrer'> bevorstehenden Update korrigiert</a>."

```

Comment:
The first link (to https://github.com/corona-warn-app/cwa-app-ios/issues/1497) in this list is not fixed.